Decred Wallet/Features/Navigation Menu/CustomTabMenuView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  CustomTabMenuView.swift
+//  Decred Wallet
+//
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+class CustomTabMenuView: UIStackView {
+    var itemTapped: ((_ index: Int) -> Void)?
+    var activeTabIndex: Int = 0
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    required init(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    convenience init(menuItems: [MenuItem], frame: CGRect) {
+        self.init(frame: frame)
+        self.axis = .horizontal
+        self.layer.backgroundColor = UIColor.white.cgColor
+        
+        for menuItem in 0 ..< menuItems.count {
```

Comment:
I actually recommended `for menuItem in menuItems`. You don't need the indices.

Reply:
resolved in 561e660